build_msvc/msvc-autogen.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import os
+import re
+
+SOURCE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src'))
+
+libs = [
+    'libbitcoin_cli',
+    'libbitcoin_common',
+    'libbitcoin_crypto',
+    'libbitcoin_server',
+    'libbitcoin_util',
+    'libbitcoin_wallet',
+    'libbitcoin_zmq',
+]
+
+ignore_list = [
```

Comment:
Looks like the reason for the ignore list is to prevent conflicts between .obj file names. But ignoring these files is not sufficient if the code in these files actually needs to be linked. Not sure if this issue affects anybody else, but I have a fix for it in 06d939c42bd6abf5211c159c6e5708a33ffe0077 from #10973.

Reply:
Thanks, that's a pretty good improvement. would allow 2 cpp files with same filename.